sdk/core/Azure.Core.Experimental/src/Amqp/AmqpMessageHeader.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core.Amqp
+{
+    /// <summary>
+    /// Represents an AMQP message header.
+    /// http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-header
+    /// </summary>
+    public class AmqpMessageHeader
+    {
+        /// <summary>
+        /// Initializes a new <see cref="AmqpMessageHeader"/> instance.
+        /// </summary>
+        public AmqpMessageHeader() { }
+
+        internal AmqpMessageHeader(AmqpMessageHeader header)
+        {
+            Durable = header.Durable;
+            Priority = header.Priority;
+            TimeToLive = header.TimeToLive;
+            FirstAcquirer = header.FirstAcquirer;
+            DeliveryCount = header.DeliveryCount;
+        }
+
+        /// <summary>
+        /// The durable value from the AMQP message header.
```

Comment:
nit:  "...from the AMQP message header" seems a bit redundant here, as this is the `AmqpMessageHeader` type.   Are there any quick copy/paste snippets we could steal from the AMQP spec here to give a better contextual understanding of what the data represents?

Reply:
maybe "AMQP transport header"